regression-v1x10.sh

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
+
+# RTE_IP alone should be enough for running the regression.
+# If for some reason, Sonoff/PiKVM IPs are not correct, they
+# should be changed in the variables.robot file
+RTE_IP=192.168.10.55
+
+# This one must be retrieved manually from the DUT before starting regression
+DEVICE_IP=192.168.10.39
+
+# Uncomment one of these
+CONFIG="protectli-v1210"
+# CONFIG="protectli-v1410"
+# CONFIG="protectli-v1610"
+
+FW_FILE="protectli_v1210_v0.9.2.rom"
+
+OS=ubuntu
+# OS=windows
+
+if [ ! -f "${FW_FILE}" ]; then
+    echo "${FW_FILE} not found. Please provide correct FW_FILE value"
+    exit 1
+fi
+
+# Function to execute the robot command
+execute_robot() {
+    local _category=$1
+    local _test_name=$2
+    local _log_file="${_test_name}_log.html"
+    local _report_file="${_test_name}_report.html"
+    local _output_file="${_test_name}.xml"
+
+    robot -L TRACE \
+          -l ${CONFIG}-${OS}/${_log_file} \
+          -r ${CONFIG}-${OS}/${_report_file} \
+          -o ${CONFIG}-${OS}/${_output_file} \
+          -v device_ip:${DEVICE_IP} \
+          -v rte_ip:${RTE_IP} \
+          -v config:${CONFIG} \
+          -v snipeit:no \
+          -v fw_file:${FW_FILE} \
+          dasharo-${_category}/${_test_name}.robot
+}
+
+# Function to handle Ctrl+C
+handle_ctrl_c() {
+    echo "Ctrl+C pressed. Exiting."
+    # You can add cleanup tasks here if needed
+    exit 1
+}
+
+# Trap SIGINT (Ctrl+C)
+trap 'handle_ctrl_c' SIGINT
+
+compatibility_tests=(
+  "efi"
+  "display-ports-and-lcd-support"
+  "usb-hid-and-msc-support"
+  # "uefi-shell"
+  "dmidecode"
+  "custom-boot-menu-key"
+  "wifi-bluetooth-support"
+  # "audio-subsystem"
+  "nvme-support"
+  # "reset-to-defaults"
+  "network-boot-utilities"
+  "eMMC-support"
+  "usb-boot"
+  "usb-detect"
+  "custom-network-boot-entries"
+  "miniPCIe-slot-verification"
+)
+
+security_tests=(
+  "tpm-support"
+  # "measured-boot"
+  # "verified-boot"
+  # "bios-lock"
+  "secure-boot"
+  # "usb-stack"
+  # "smm-bios-write-protection"
```

Comment:
```suggestion
  "smm-bios-write-protection"
```

Reply:
Done: 5d47050dbc62a66b0155665c81545b157b319673

Reply:
Protectli decided to not have this feature, so we have to revert it.

Reply:
This is a better way and will allow us to unify the logs with the results.
It would be useful to indicate where to set the variables:
`RTE_IP, DEVICE_IP, CONFIG, FW_FILE`

I will close this PR without merging.